ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/error/ErrorInterceptorSpec.scala

```diff
@@ -162,7 +160,46 @@ final class ErrorInterceptorSpec
             }
         }
       }
+    }
+  }
+
+  LogOnUnhandledFailureInClose.getClass.getSimpleName - {
+    "is transparent when no exception is thrown" in {
+      @volatile var idx = 0
+      val call = () => {
+        idx += 1
```

Comment:
I think `volatile` is not enough to make getAndIncrement operation thread safe. Probably an AtomicInteger is needed